### ISSUE TITLE
constellations_list how to?

### ISSUE THREAD

**Issue by commentsonly** [partofthething/ha_skyfield#7]
This is a nice one no question.
Since show_constellations: true resulted in heavy load I wanted to see how it might perform if only usra major being allowed as the one and only constellation. I would have continued evaluating if then adding ursa major would work aswell. Since 2-3 constellations surely make it somewhat easier to help around getting the correct location of a planet.
But I didn't manage to get it going. Any configuration help on constallations_list?
TIA

**Comment by partofthething** [comment]
Good question. Yeah I should have put an example. The list works with the names from [the data source](https://github.com/partofthething/ha_skyfield/blob/master/custom_components/ha_skyfield/constellations_by_RA_Dec.dat). 

The default list in the code looks like: `DEFAULT_CONSTELLATIONS = ZODIAC + ["Cassiopeia", "Orion", "Pegasus", "UrsaMajor"]` so to just get UrsaMajor you'd use something like:

```
camera:
    platform: ha_skyfield
    show_constellations: true
    constellations_list: 
        - UrsaMajor
```
should work.

**Comment by commentsonly** [comment]
:-) dammit. I tried with space, all lowercase, I even tried 'UMa' since I found thatin the lib specs.
I'll give it a try and thanks for the link to the data source. (hopefully you added this to the main description aswell.

Besides that I just fancy a tripplicated legend (3 times all the planets in differnt sort order. any idea?

**Comment by partofthething** [comment]
Closed with 065a918a58c4
